src/plugins/autoRowSize/autoRowSize.js

```diff
@@ -12,6 +12,7 @@ import { PhysicalIndexToValueMap as IndexToValueMap } from './../../translations
 
 const ROW_WIDTHS_MAP_NAME = 'autoRowSize';
 
+/* eslint-disable jsdoc/require-description-complete-sentence */
 /**
  * @plugin AutoRowSize
  *
```

Comment:
```suggestion
 * @class AutoRowSize
```
The _AutoRowSize_ plugin needs to be updated